src/media/js/addon/components/addon.js

```diff
@@ -97,6 +109,8 @@ export class Addon extends React.Component {
           <dd>{this.props.version}</dd>
         </dl>
 
+        {this.renderDescription()}
```

Comment:
Right, but you couldn't do this if you wanted:

    renderDescription() {
      if (this.props.description) {
        return (
          <dt>Description:</dt>
          <dd>{this.props.description}</dd>
        );
      }
    }